README.md

```diff
@@ -186,3 +186,53 @@ Lint errors will be displayed on the console while the app is running.
 
 The file `ColorScheme.css` defines the colors used in the project.
 Additionally, there is one CSS file for each component.
+
+
+## Using towncrier to automatically produce the changelog
+To install towncrier using `pip` run
+```console 
+$ pip install towncrier
+```
```

Comment:
No, not really, since towncrier is a python library, so python is required to run it.

Reply:
But technically towncrier is not required to create the news fragments, that can also be done manually. 
But for creating the CHANGELOG it is needed

Reply:
I think it would be nice to keep towncrier, so that we don't have multiple release note tools that we need to keep track of. 

The check runs in Github actions, where we have `pip` available and that check only checks if a news fragment (a file in the folder `changelog.d`) exists, it does not matter if that file was created using towncrier or any other way.
So the only place we actually *need* to have towncrier installed locally is when making releases, where I think we can assume to have python available, or can we not assume that?

Reply:
The changelog becomes the release manager's responsibility, and nobody else really needs to touch towncrier as long as they are able to produce compliant news fragments using their preferred edit.

Who is the release manager? @hmpf. I don't think she has a problem with towncrier.  I really don't see the need to research other tools to further complicate our team's stack :)

Reply:
Regarding actual ways to install towncrier: `pip` is the standard tool. `pipx` could be recommended if you don't want to fiddle with virtual environments and can't/won't use `pip` to install to system directories.

Other than that, `towncrier` is packaged for Debian Bookworm and and has been available in Ubuntu since 22.04LTS.